src/components/spreadsheet/table-wrapper.js

```diff
@@ -139,25 +146,10 @@ const TableWrapper = (props) => {
         [lockedColumnsNames.size]
     );
 
-    //the following variable needs to be a ref because its usage in EditingCellRenderer sets and reads
-    //the value although it is not rerendered so storing it in a state wouldn't fill its purpose
-    const isValidatingData = useRef(false);
```

Comment:
I'm wondering, you were able to remove isValidatingData because we no longer care if the editors are closed when the user clicks away ? Or is there another reason